src/core/components/googleButton.component.tsx

```diff
@@ -31,23 +47,38 @@ const GoogleButtonComponent: React.FC<ButtonProps> = props => {
         size: "large",
         shape: "pill",
         logo_alignment: "left",
-        type: "standard"
+        type: "icon"
       })
     }
     initGoogleButton()
   }, [activeSnackBar, history, loginGoogle])
 
-  return <div id="google_signin" style={{ width: "100%" }} />
+  const handleClick = useCallback((e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+    e.stopPropagation()
+
+    const el = document.querySelectorAll('[role="button"]')[0] as HTMLElement
+    el.click()
+  }, [])
+
+  return (
+    <>
+      <div id="google_signin" style={{ width: "100%", display: "none" }} />
+      <Button onClick={handleClick} className={`${classes.button} ${props.className}`} {...props}>
+        <img src={googleIcon} alt="googleIcon" />
+        <Typography variant="button" className={classes.capitalText}>
+          เข้าสู่ระบบด้วยบัญชี Google
+        </Typography>
+      </Button>
```

Comment:
should we use `label` tag for observing google button?

Reply:
Would you mind give some additional explaination

Reply:
nvm. It can't be used with `div` elements.

https://stackoverflow.com/questions/11705956/can-the-label-tags-for-attribute-be-associated-with-a-normal-div

Reply:
อะไรประมาณนี้. ด้วยท่านี้เราไม่ต้องทำ `handleClick` เองก็ได้

https://user-images.githubusercontent.com/33742791/141605806-c73fd9cb-5f9b-4a98-ae21-b214e6e0cf62.mov

Reply:
It's not working too I found that the onClick handler is actually binding to div element which have attribute role="button"

https://user-images.githubusercontent.com/35169079/141605932-1e2b1ba1-87ea-42be-8c1d-5123233ef73a.mp4

Reply:
It's inside two elements the div (which sometimes not appear) or iframe
![image](https://user-images.githubusercontent.com/35169079/141605978-bb4e69b1-70e3-4b50-9e0d-7f933939053c.png)

Reply:
OK